cmd/generate.go

```diff
@@ -0,0 +1,171 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+// generateCmd represents the generate command
+var generateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate docker-compose.yml and harbor-compose.yml files from an existing shipment",
+	Long: `
+$ harbor-compose generate my-shipment dev
+or
+$ harbor-compose generate my-shipment qa
+`,
+	Run: generate,
+}
+
+func init() {
+	RootCmd.AddCommand(generateCmd)
+}
+
+func generate(cmd *cobra.Command, args []string) {
+	if len(args) < 2 {
+		log.Fatal("2 arguments are required. ex: harbor-compose generate my-shipment dev")
+	}
+
+	shipment := args[0]
+	env := args[1]
+
+	if Verbose {
+		log.Printf("fetching shipment...")
+	}
+	shipmentObject := GetShipmentEnvironment(shipment, env)
```

Comment:
this function needs to always ask for authentication or get a token and username passed into it.